app/views/static_pages/about.html.haml

```diff
@@ -83,6 +83,6 @@
       %li
         Developers are welcome to join the project.  The
 
-        = link_to "README document"
+        = link_to "README document", "https://github.com/swapmyvote/swapmyvote/blob/master/README.md"
```

Comment:
Wow good catch, @tdg do we have a Google Search Console or similar set up to catch broken links?

Reply:
I don't think we do, no. I will have a look into it tomorrow